windows/src/desktop/kmshell/install/uninstall.pas

```diff
@@ -99,7 +99,7 @@ function UninstallPackage(Owner: TWinControl; const PackageName: WideString; Sil
     pkg := nil;
     if WaitForElevatedConfiguration(Handle, '-up "'+pkgID+'" -s') = 0 then
     begin
-      kmcom.Keyboards.Refresh;
+      kmcom.Refresh;
       kmcom.Keyboards.Apply;
```

Comment:
For example, why do we use `kmcom.Refresh` but not `kmcom.Apply` here, instead using `kmcom.Keyboards.Apply`?

Reply:
This should probably be `kmcom.Apply`, matching the code changes further down. I think I just missed it.

```suggestion
      kmcom.Refresh;
      kmcom.Apply;
```